docs/CLIENT.md

```diff
@@ -510,3 +517,8 @@ client = new SphereClient
 }
 
 ```
+
+#### Correlation Header ID
+Each response from the HTTP API contains a special header `x-correlation-id`. This is specially useful for the SPHERE.IO team in order to better correlate failure requests with other possible causes.
```

Comment:
i would remove one of the "special"/"specially" in the description. Maybe something like this:

"This unique value can be used to correlate events across different layers and also might help in case of failures."